napari/layers/labels/_tests/test_labels.py

```diff
@@ -1110,6 +1112,11 @@ def test_large_label_values():
     assert len(np.unique(mapped.reshape((-1, 4)), axis=0)) == 4
 
 
+@pytest.mark.xfail(
+    parse_version(version('zarr')).is_prerelease,
+    reason='tensorstore is not compatible with zarr 3',
+    strict=True,
+)
```

Comment:
As I understand it, this is a problem with the zarr v3 format rather than zarr-python v3, in which case, perhaps we can change the test to write a zarr v2 file?

Reply:
https://zarr.readthedocs.io/en/v3/_autoapi/zarr/index.html#zarr.open

using `zarr_version=2` should work

Reply:
it looks working now